_posts/2020-05-18-using-git-hooks-with-flyway.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: blog
+subtitle: Using git hooks with Flyway
+permalink: /blog/using-git-hooks-with-flyway.html
+author: julia
+---
+
+When you're on your own developing a simple application, managing your migration scripts is not a demanding task; each
+time you add a script, it gets the next version number in whatever sequence you've decided on using. When your
+application takes off and you find you have a team working on migrations simultaneously, sharing code via a
+distributed source control system such as `git`, then there is an increased risk of running into problems as the 
+developers work on the same database migration code at the same time.
+
+## Problem one: Numbering conflicts
+
+Let's say you've been asked to add a new feature to your application, and this needs a new column to be added to the
+database. The database schema is currently at v4; so the natural thing to do is create a script `V5__MyNewColumn.sql`
+and check it into source control. So far, all is good.
+
+Meanwhile, one of your colleagues is also working on a new feature and has added a script `V5__MyNewTable.sql`. As far
+as he's concerned, all appears good too. When the two sets of changes make it into the master branch ready
```

Comment:
Should we be using gender neutral pronouns? e.g. As far as they're concerned

Reply:
:can-of-worms:

I think whatever we go with it should be consistent throughout; there's quite a few "he" which would also need to change.

Reply:
Actually, just side-stepped it by making colleagues plural (so no pronoun issue).